examples/onoff_light/src/main.rs

```diff
@@ -74,6 +74,7 @@ fn run() -> Result<(), Error> {
         device_name: "OnOff Light",
         product_name: "Light123",
         vendor_name: "Vendor PQR",
+        unique_id: "aabbccdd",
```

Comment:
Per my analysis, this actually isn't required.  We can choose to make this an Option<String> and interpret None as "" and it will both work and be consistent with the main matter repo behaviour.